doc/website/blog/2023-12-11-gitops-for-databases-part-2.mdx

```diff
@@ -0,0 +1,309 @@
+pod/argocd-applicationset-controller-69dbc8585c-6qbwr condition met
+pod/argocd-dex-server-59f89468dc-xl7rg condition met
+pod/argocd-notifications-controller-55565589db-gnjdh condition met
+pod/argocd-redis-74cb89f466-gzk4f condition met
+pod/argocd-repo-server-68444f6479-mn5gl condition met
+pod/argocd-server-579f659dd5-5djb5 condition met
+```
+
+For more information or if you run into some error refer to the
+[Argo CD Documentation](https://argo-cd.readthedocs.io/en/stable/getting_started/).
+
+### 2. Install the Atlas Operator
+
+```bash
+helm install atlas-operator oci://ghcr.io/ariga/charts/atlas-operator
+```
+
+Helm will print something like this:
+
+```bash
+Pulled: ghcr.io/ariga/charts/atlas-operator:0.3.6
+Digest: sha256:7e29c15e846fa9c25164f4ad5a7cb7f25e9ead2882082f0352985e58c1976f99
+NAME: atlas-operator
+LAST DEPLOYED: Mon Dec 11 10:25:11 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Wait until the atlas-operator pod is running:
+
+```bash
+kubectl wait --for=condition=ready pod -l app.kubernetes.io/name=atlas-operator -n default
+```
+
+`kubectl` will print something like this:
+
+```bash
+pod/atlas-operator-866dfbc56d-qkkkn condition met
+```
+
+For more information on the installation process refer to the [Atlas Operator Documentation](/integrations/kubernetes/operator#getting-started)
+
+### Step 2: Set up the Target Database
+
+Start by deploying a simple PostgreSQL database using the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/ariga/atlas-operator/master/config/integration/databases/postgres.yaml
+```
+
+This will create a `Deployment` which runs a single (non-persistent) PostgreSQL instance and a `Service` that
+exposes it on port `5432`. In addition, it will create a `Secret` that contains the database credentials.
+
+Wait until the database pod is running:
+
+```bash
+kubectl wait --for=condition=ready pod -l app=postgres -n default
+```
+
+### Step 3: Create the `AtlasMigration` resource
+
+In order for the Atlas Operator to know which migrations to apply, we need to create an `AtlasMigration` resource
+that points to the Atlas Cloud project we created in [part one](2023-12-06-gitops-for-databases-part-1.mdx).
+Create a new directory called `manifests` in your GitHub repository with the following contents. In it,
+create a file called `atlas-migration.yaml` with the following contents:
+
+```yaml title="manifests/atlas-migration.yaml"
+apiVersion: db.atlasgo.io/v1alpha1
+kind: AtlasMigration
+metadata:
+  name: migration
+spec:
+  urlFrom:
+    secretKeyRef:
+      key: url
+      name: postgres-credentials
+  cloud:
+    project: "atlasdemo" # Atlas Cloud project name
+    tokenFrom:
+      secretKeyRef:
+        name: atlas-credentials
+        key: token
+  dir:
+    remote:
+      name: "atlasdemo" # Migration directory name in your atlas cloud project
+      tag: "1d579be616db48803bb21713fd836a9165030f18"
```

Comment:
This isn't static, is it?

Reply:
I read now the below. I think you should add a comment here maybe, that yo will explain how to obtain the tag. Because I'd be confused if I stumble across this.